pkgs/tools/misc/ffsend/default.nix

```diff
@@ -54,6 +61,6 @@ buildRustPackage rec {
     homepage = https://gitlab.com/timvisee/ffsend;
     license = licenses.gpl3;
     maintainers = [ maintainers.lilyball ];
-    platforms = platforms.darwin ++ platforms.linux;
+    platforms = platforms.darwin ++ platforms.linux ++ [ stdenv.lib.systems.inspect.patterns.isBSD ];
```

Comment:
I think we can use
```suggestion
    platforms = platforms.unix;
```

Reply:
As I mentioned in the PR description, that includes Cygwin and I don't know if `ffsend` supports Cygwin. I wasn't sure if it was better to just assume it functions under Cygwin in order to avoid this awkward platforms expression.

Reply:
After thinking about it, I've gone ahead and applied this suggestion.